pkg/cluster/config.go

```diff
@@ -340,3 +353,16 @@ func IsCloudControllerOwnedByCCM(oclient configclient.Interface) (bool, error) {
 
 	return ownedByCCM, nil
 }
+
+// getProxyVars returns a map of the proxy variables and values from the WMCO container's environment. The presence of
+// any implies a proxy is enabled, since OLM would inject into the operator spec. Returns an empty map otherwise.
+func getProxyVars() map[string]string {
+	proxyVarsMap := make(map[string]string, len(acceptableProxyVars))
+	for _, envVar := range acceptableProxyVars {
+		value, found := os.LookupEnv(envVar)
```

Comment:
Is this run on the WMCO pod? Do we need to do anything in the operator manifests to get CNO to inject these variables into the pod?

Reply:
Yes, the lookup is run on the WMCO pod.

OLM injects the vars into the pod env btw, not CNO. CNO injects the CA bundle into the trusted-ca configmap. We don't need anything in the manifests to enable OLM's injection, it does so to all it's managed operators automatically

Reply:
Please add a comment.

Reply:
added to the function doc and comment